src/components/utils/weatherVariables.jsx

```
@@ -353,4 +355,103 @@ export const ShowWeather = (props) => {
+                      color="black"
+                  />
+              )}
+              <span className="text-2xl ml-2">{Math.round(currentLocationWeather.main?.temp)}°C</span>
+          </div>
+          <p className="capitalize">{currentLocationWeather.weather && currentLocationWeather.weather[0]?.description}</p>
+          <div className="mt-2 grid grid-cols-2 gap-2 text-sm">
+              <div>Humidity: {currentLocationWeather.main?.humidity}%</div>
+              <div>Wind: {Math.round(currentLocationWeather.wind?.speed)} m/s ({<WindForce windSpeed={currentLocationWeather.wind?.speed} />})</div>
+              <div>Pressure: {currentLocationWeather.main?.pressure} hPa</div>
+              <div>Visibility: {currentLocationWeather.visibility >= 1000 ? `${currentLocationWeather.visibility / 1000} km` : `${currentLocationWeather.visibility} m`}</div>
+              <div>Sunrise: {(localSunriseSunsetTimes?.sunriseHour > 23 ? localSunriseSunsetTimes.sunriseHour - 24 : localSunriseSunsetTimes?.sunriseHour < 0 ? localSunriseSunsetTimes?.sunriseHour + 24 : localSunriseSunsetTimes?.sunriseHour).toString().padStart(2, '0')}:{localSunriseSunsetTimes?.sunriseMinute.toString().padStart(2, '0')}</div>
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)